.gitignore

```diff
@@ -1,5 +1,7 @@
 # ## Linux
-.*
+.cache
+.coverage
+build/
 !.gitignore
 /dist/
```

Comment:
this should be probably 'dist/'

Reply:
Slash in front of each item means only ignore starting at root of repo. It should probably stays with leading slash, and added to the other items too :)